### PR TITLE
Add mastodon icon

### PR DESCRIPTION
**Pull request by ken-salterdiazsolutions**
I added a mastodon icon.  Because of the federated nature, you have to specify the "network" attribute to "mastodon".

### CODE REVIEW

File: src/_networks-db.js [hunk 1]
```diff
@@ -169,6 +169,12 @@ export default {
...
...1.669222 41.394264 25.113281 39.900391 C 25.111831 40.232807 25.135091 40.566738 25.181641 40.896484 C 25.6893 44.442265 28.949007 44.655733 32.042969 44.755859 C 35.166023 44.849311 37.689453 44.033203 37.689453 44.033203 L 37.818359 46.671875 C 37.818359 46.671875 35.894996 47.76521 32.003906 47.966797 C 29.859806 48.080272 27.193032 47.917303 24.091797 47.152344 C 15.994012 45.143157 16 34.753512 16 26.480469 C 16 19.530418 20.882812 17.488281 20.882812 17.488281 C 23.152736 16.527743 27.516498 16.01635 31.910156 16 z M 27.207031 21.300781 C 25.701689 21.263734 24.179815 21.809385 23.191406 22.832031 C 21.389141 24.750438 21.785156 26.005597 21.785156 34.648438 L 25.353516 34.648438 L 25.353516 27.134766 C 25.353516 23.617019 30.236328 23.483806 30.236328 27.625 L 30.236328 31.978516 L 33.773438 31.978516 L 33.773438 27.625 C 33.773438 23.483805 38.660156 23.618973 38.660156 27.136719 L 38.660156 34.652344 L 42.220703 34.652344 L 42.220703 34.648438 C 42.220703 26.012272 42.623991 24.765123 40.814453 22.832031 C 38.85218 20.800149 34.726804 20.635758 32.886719 23.261719 L 32.003906 24.644531 L 31.117188 23.261719 C 30.200781 21.956081 28.712374 21.337831 27.207031 21.300781 z ',
```
Comment (tguichaoua) [inline src/_networks-db.js:175]: There is a trailing space in the string.

```suggestion
    mask: 'M -0.13476562 -0.13476562 L -0.13476562 64.40625 L 64.542969 64.40625 L 64.542969 -0.13476562 L -0.13476562 -0.13476562 z M 31.910156 16 C 36.303815 15.983647 40.72947 16.462327 43.123047 17.488281 C 43.123047 17.488281 47.998047 19.530418 47.998047 26.480469 C 47.998047 26.480469 48.062124 31.606168 47.314453 35.167969 C 46.837341 37.428137 43.095755 39.901591 38.798828 40.380859 C 37.259848 40.55174 31.669222 41.394264 25.113281 39.900391 C 25.111831 40.232807 25.135091 40.566738 25.181641 40.896484 C 25.6893 44.442265 28.949007 44.655733 32.042969 44.755859 C 35.166023 44.849311 37.689453 44.033203 37.689453 44.033203 L 37.818359 46.671875 C 37.818359 46.671875 35.894996 47.76521 32.003906 47.966797 C 29.859806 48.080272 27.193032 47.917303 24.091797 47.152344 C 15.994012 45.143157 16 34.753512 16 26.480469 C 16 19.530418 20.882812 17.488281 20.882812 17.488281 C 23.152736 16.527743 27.516498 16.01635 31.910156 16 z M 27.207031 21.300781 C 25.701689 21.263734 24.179815 21.809385 23.191406 22.832031 C 21.389141 24.750438 21.785156 26.005597 21.785156 34.648438 L 25.353516 34.648438 L 25.353516 27.134766 C 25.353516 23.617019 30.236328 23.483806 30.236328 27.625 L 30.236328 31.978516 L 33.773438 31.978516 L 33.773438 27.625 C 33.773438 23.483805 38.660156 23.618973 38.660156 27.136719 L 38.660156 34.652344 L 42.220703 34.652344 L 42.220703 34.648438 C 42.220703 26.012272 42.623991 24.765123 40.814453 22.832031 C 38.85218 20.800149 34.726804 20.635758 32.886719 23.261719 L 32.003906 24.644531 L 31.117188 23.261719 C 30.200781 21.956081 28.712374 21.337831 27.207031 21.300781 z',
```

**Review by tguichaoua — APPROVED** [review]
LGTM

**Comment by clarmso** [comment]
Please make mastodon 🐘 icon available soon!

**Comment by couetilc** [comment]
@ken-salterdiazsolutions this was released in [v6 of react-social-icons](https://www.npmjs.com/package/react-social-icons)

CC @tguichaoua

**Pull request closed without merging**